hydrolib/core/io/ini/models.py

```diff
@@ -64,17 +58,13 @@ class Section(BaseModel):
 
     Attributes:
         header (str): The header (without brackets) of this Section
-        start_line (int): The starting line index of this Section block
-        end_line (int): The end line index of this Section block
         content (List[Union[Property, CommentBlock]]):
             The ordered list of Property and CommentBlock objects
         datablock (Optional[Sequence[Sequence[str]]]):
```

Comment:
Maybe document what the "dimentions" are? e.g. whether it is rows x colummns or columns x rows.